backend/danswer/server/manage/administrative.py

```diff
@@ -231,3 +235,56 @@ def update_token_budget_settings(
     # Store the settings in the dynamic config store
     get_dynamic_config_store().store(TOKEN_BUDGET_SETTINGS, settings_json)
     return {"message": "Token budget settings updated successfully."}
+
+
+@router.put("/admin/users")
+def bulk_invite_users(
+    _: User | None = Depends(current_admin_user),
+    emails: list[str] = Body(..., embed=True),
+) -> int:
+    all_emails = list(set(emails) | set(get_invited_users()))
+    return write_invited_users(all_emails)
+
+
+@router.patch("/admin/remove-invited-user")
+def remove_invited_user(
+    user_email: UserByEmail,
+    _: User | None = Depends(current_admin_user),
+) -> int:
+    users = get_invited_users()
```

Comment:
nit: for readability so reviewers don't need to check the function, users generally refer to User objects, in this case they're user_emails which are strings.